enterprise/frontend/src/embedding-sdk/package.template.json

```
@@ -8,6 +8,17 @@
   },
   "license": "SEE LICENSE IN LICENSE.txt",
   "main": "./dist/main.bundle.js",
+  "exports": {
+    ".": {
+      "types": "./dist/enterprise/frontend/src/embedding-sdk/index.d.ts",
+      "default": "./dist/main.bundle.js"
+    },
+    "./next": {
+      "types": "./dist/enterprise/frontend/src/embedding-sdk/next.d.ts",
```

This seems to be the incorrect path.

```suggestion
      "types": "./dist/next.d.ts",
```

I tried keeping this path, and write the file to `dist/enterprise/frontend/src/embedding-sdk/next.d.ts` instead, but it came with a bunch of path problems, so I think fixing this one is easier.

---

Thanks for catching that, it seems that types are still somewhat broken even with that.
They do work for the first "layer", like it sees that MetabaseProvider is a react component, but then it can't resolve SDKConfig and some other types, I'll investigate it after I fix https://github.com/metabase/metabase/issues/50279